.coderabbit.yml

```
@@ -12,24 +12,23 @@ reviews :
+        - Validate inputs using `@Valid` along with `@NotBlank` or `@NotNull`.
+        - Use positive names for validation and boolean-returning methods (e.g., `isValid`, `hasPermissions`).
+        - Use `request` or `response` instead of the general `dto` naming.
+        - Use a trailing comma when defining enum values.
+        - Use `@Getter`, `@NoArgsConstructor`, `@RequiredArgsConstructor`, and `@Builder` (for 4+ fields) instead of `@Data`.
+        - Place important annotations like `@Entity` or `@Table` above Lombok annotations.
+        - Create DTOs using `record` classes.
+        - Order methods with `public` methods first, and `private` methods directly below the `public` method that calls them.
+        - Do not declare uninitialized variables.
+        - Explicitly define `nullable = false` in the `@Column` annotation.
+        - Avoid using `@RequestMapping` at the class level in a `@RestController`.
+        - Apply `@Transactional(readOnly = true)` at the top of the service class.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)